core/src/com/testing/objects/logic/Sequence.java

```diff
@@ -0,0 +1,154 @@
+package com.testing.objects.logic;
+
+import com.testing.BaseObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Data and basic functionality for {@link GameLogic}
+ */
+public class Sequence extends BaseObject {
```

Comment:
- is there a reason that sequence needs to extend BaseObject? 
- is there a reason the sequence is a seperate class and not a part of GameLogic?

Reply:
* `Sequecne` no longer a `BaseObject`, moved the method which required it to be one to `GameLogic`
* `Sequence` is its own class because `GameLogic` was getting too big, there is no better reason for this.